src/containers/linkCreator/linkCreator.jsx

```diff
@@ -0,0 +1,123 @@
+// @flow
+import React, { Component } from "react";
+import styled from "styled-components";
+import _ from "lodash";
```

Comment:
Don't use lodash, it is very heavy library. It could increase prod bundle. All methods that we have lodash exists in js.